app/services/items_service.rb

```diff
@@ -0,0 +1,20 @@
+class ItemsService
+  class << self
+
+    def call_api(path)
+      response = conn.get(path)
+      parse_data(response)
+    end
+
+    private
+
+    def conn
+      Faraday.new(url: "http://localhost:3000")
+    end
+
+    def parse_data(response)
+      JSON.parse(response.body, symbolize_names: true)
+    end
+
+  end
+end
```

Comment:
You could combine the ITemService and the MerchantService since both services are calling to the same service (the rails_engine project api). You can have one service per api that you're using.